omero/sysadmins/version-requirements.rst

```diff
@@ -842,24 +913,14 @@ If no version is provided, a suitable repository is indicated.
       - Ubuntu
       - Homebrew
       - FreeBSD Ports
-    * - 1.6
-      - N/A
-      - N/A
-      - N/A
-      - N/A
-    * - 1.8
-      - N/A
-      - N/A
-      - N/A
+    * - 1.12
+      - 7 (`EPEL <https://dl.fedoraproject.org/pub/epel/7/x86_64/>`__)
+      - 14.04 (`nginx <https://launchpad.net/~nginx/+archive/ubuntu/stable>`__)
       - N/A
-    * - 1.10
-      - 6 (`EPEL <https://dl.fedoraproject.org/pub/epel/6/x86_64/>`__), 7 (`EPEL <https://dl.fedoraproject.org/pub/epel/7/x86_64/>`__)
-      - 14.04 (`nginx <https://launchpad.net/~nginx/+archive/ubuntu/stable>`__), 16.04
-      - Yes
       - Yes
-    * - 1.12
-      - 6 (`EPEL <https://dl.fedoraproject.org/pub/epel/6/x86_64/>`__), 7 (`EPEL <https://dl.fedoraproject.org/pub/epel/7/x86_64/>`__)
-      - 14.04 (`nginx <https://launchpad.net/~nginx/+archive/ubuntu/stable>`__), 16.04
+    * - 1.14
+      - N/A
```

Comment:
What does the N/A mean here? If it means, no repo is needed, then either "Yes" from this table or perhaps "7" from the Java table would be appropriate. I'm a bit confused.

Reply:
I don't know what it means but it pervades the existing Homebrew columns. As far as I could see the pattern seemed to be that it tends to follow old versions, hence my guess here, though I wonder if one could go back in Homebrew's history to resurrect them. Would certainly be glad for somebody who knows about Homebrew stuff to either guide or fix; could be a separate PR.